src/jsd/platinum_actuator_offline.h

```diff
@@ -1,20 +1,20 @@
-#ifndef FASTCAT_EPD_ACTUATOR_OFFLINE_H_
-#define FASTCAT_EPD_ACTUATOR_OFFLINE_H_
+#ifndef FASTCAT_PLATINUM_ACTUATOR_OFFLINE_H_
+#define FASTCAT_PLATINUM_ACTUATOR_OFFLINE_H_
 
 // Include related header (for cc files)
 
 // Include c then c++ libraries
 
 // Include external then project includes
-#include "fastcat/jsd/epd_actuator.h"
+#include "fastcat/jsd/platinum_actuator.h"
 
 namespace fastcat
 {
-// ActuatorOffline simulates both EgdActuator and EpdActuator.
-class EpdActuatorOffline : public EpdActuator
+// ActuatorOffline simulates both GoldActuator and PlatinumActuator.
```

Comment:
It seems that I forgot to remove this comment. It does not apply anymore because now the offline classes of the Gold and Platinum drives are separate.